andela/travel_tool_front
# #160371607 Checkbox status should be cached

#### What does this PR do?
This PR ensures that if the checkbox to remember user personal details is checked, the details will be retrieved including checkbox status

#### Description of Task to be completed?
Currently, the checkbox is unchecked on initiation of the next request, however, if checked, the checkbox should be checked on initiation of the next request and all the personal information returned in an uneditable format. On the other hand, if it is unchecked it should remain unchecked on initiation of next request.


#### How should this be manually tested?
* Clone this repo branch  - `git clone -b bg-checkbox-state-cached-160371607 https://github.com/andela/travel_tool_front.git`
* Switch to the repo directory - cd travel_tool_front
* Install dependencies and start the server - make start
* Set up the server by following this guide
* Navigate to localhost:5000 in a browser
* sign in using your andela email address
* Click the `New request` button
* Add your personal details, check the checkbox and submit form
* Navigate back to the `New request` button
* the checkbox should be checked



#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#160371607](https://www.pivotaltracker.com/story/show/160371607)


#### Snapshots
![image](https://user-images.githubusercontent.com/30747298/45805641-d6058580-bcc7-11e8-8fc9-0a0ca84569ba.png)

![image](https://user-images.githubusercontent.com/30747298/45805762-16fd9a00-bcc8-11e8-99fe-54708aa636d7.png)



#### Questions:
